src/server/pegasus_server_impl.cpp

```diff
@@ -293,7 +293,43 @@ pegasus_server_impl::pegasus_server_impl(dsn::replication::replica *r)
     bool disable_bloom_filter = dsn_config_get_value_bool(
         "pegasus.server", "rocksdb_disable_bloom_filter", false, "Whether to disable bloom filter");
     if (!disable_bloom_filter) {
-        tbl_opts.filter_policy.reset(rocksdb::NewBloomFilterPolicy(10, false));
+        // average bits allocated per key in bloom filter.
+        // bits_per_key    |           false positive rate
```

Comment:
Why would `false positive rate` greater than 1?

Reply:
Here `false positive rate` in percentage.